app/src/main/java/xyz/jhughes/laundry/activities/LocationActivity.java

```diff
@@ -37,18 +40,27 @@
     @Bind(R.id.location_activity_toolbar) Toolbar toolbar;
     @Bind(R.id.progressBar) ProgressBar mLoadingProgressBar;
 
-
     private LocationAdapter adapter;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
```

Comment:
Do you need to make a check here to make sure that this is a cold open of the app? Also clear the lastRoom pref once you do this.